x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/downsample/DownsampleAction.java

```diff
@@ -20,40 +20,39 @@
 import org.elasticsearch.tasks.TaskId;
 import org.elasticsearch.xcontent.ToXContentObject;
 import org.elasticsearch.xcontent.XContentBuilder;
-import org.elasticsearch.xpack.core.rollup.RollupActionConfig;
 
 import java.io.IOException;
 import java.util.Map;
 import java.util.Objects;
 
 import static org.elasticsearch.action.ValidateActions.addValidationError;
 
-public class RollupAction extends ActionType<AcknowledgedResponse> {
-    public static final RollupAction INSTANCE = new RollupAction();
-    public static final String NAME = "indices:admin/xpack/rollup";
+public class DownsampleAction extends ActionType<AcknowledgedResponse> {
+    public static final DownsampleAction INSTANCE = new DownsampleAction();
+    public static final String NAME = "indices:admin/xpack/downsample";
```

Comment:
Will this break the mixed cluster case? That's fine for the unreleased downsample stuff but not for the old rollup code I think.

Reply:
Old rollup code is behind a feature flag and has not been released. We will probably need to bump up the supported versions to 8.5.0 for those tests.